internal/auth/common/common.go

```diff
@@ -35,3 +43,19 @@ func GetIdentity(ctx context.Context) (*Identity, error) {
 	}
 	return identity, nil
 }
+
+func ExtractBearerToken(r *http.Request) (string, error) {
+	authHeader := r.Header.Get(AuthHeader)
+	if authHeader == "" {
+		return "", fmt.Errorf("empty %s header", AuthHeader)
+	}
+	token := strings.TrimPrefix(authHeader, "Bearer ")
+	if token == authHeader {
+		return "", fmt.Errorf("invalid %s header", AuthHeader)
```

Comment:
Maybe put the auth header value in the error message?

Reply:
that smells like leaking secrets

Reply:
OK